examples/text_classification/tc_multi_languages_transformers.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
do we have a test for this notebook?

Reply:
I don't have a test for this notebook. It looks that Sharat wants to delete the other 3 notebooks (MultiNLI, DAC, and BBC Hindi notebooks), and just keep this multi language one. I will only need to update the test function in one of those notebooks then.